--- gnomeicu-0.96.1/configure.in~	Wed Dec 24 23:43:44 2003
+++ gnomeicu-0.96.1/configure.in	Wed Dec 24 23:43:44 2003
@@ -162,15 +162,15 @@
 AC_ARG_WITH(
        gdbm-dir,
        [  --with-gdbm-dir=BASE-DIR         Specifies the base gdbm directory],
-       LIBDBM="-L$withval/lib -lgdbm"
-       AC_DEFINE(HAVE_LIBGDBM)
-       CPPFLAGS="$CPPFLAGS -I$withval/include"
-       libdb_parameter=yes)
+       [ LIBDBM="-L$withval/lib -lgdbm"
+         AC_DEFINE(HAVE_LIBGDBM)
+         CPPFLAGS="$CPPFLAGS -I$withval/include"
+         libdb_parameter=yes ])
 
 AC_ARG_WITH(
        ndbm,
        [  --with-ndbm=[yes|no]             Use ndbm instead of gdbm],
-        LIBDBM=""
+       [ LIBDBM=""
 
 	if test x$libdb_parameter != xno
 	then
@@ -185,12 +185,12 @@
 	           	AC_MSG_WARN(No Solaris NDBM functions found!)
 	        fi
 	        libdb_parameter=yes
-	fi
+	fi ]
 )
 
 if test x$libdb_parameter != xyes
 then
-       AC_CHECK_LIB(gdbm,gdbm_open,LIBDBM="-lgdbm",LIBDBM="")
+       AC_CHECK_LIB(gdbm,gdbm_open,[LIBDBM="-lgdbm"],[LIBDBM=""])
        if test x"" != x"${LIBDBM}"; then
                AC_DEFINE(HAVE_LIBGDBM)  
                libdb_parameter=yes
@@ -229,7 +229,7 @@
  AC_MSG_RESULT(yes)
 ])
 
-AC_CHECK_HEADER([iconv.h], AC_DEFINE(HAVE_ICONV))
+AC_CHECK_HEADER([iconv.h], [AC_DEFINE(HAVE_ICONV)])
 
 AC_OUTPUT(
 Makefile
