diff -urN gnomeicu-0.99.5.orig/GnomeICU.desktop.in gnomeicu-0.99.5.new/GnomeICU.desktop.in
--- gnomeicu-0.99.5.orig/GnomeICU.desktop.in	2003-04-16 08:07:28.000000000 +0200
+++ gnomeicu-0.99.5.new/GnomeICU.desktop.in	2004-07-16 22:52:51.781981040 +0200
@@ -3,9 +3,10 @@
 _Comment=Communicate with your friends
 Exec=gnomeicu
 Icon=gnome-gnomeicu.png
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;Network;
+Encoding=UTF-8
+Categories=GTK;GNOME;Network;InstantMessaging;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=GnomeICU
 X-GNOME-Bugzilla-Component=
